.github/workflows/publish-docker.yml

```
@@ -71,6 +75,14 @@ jobs:
         cache-from: type=gha
         cache-to: type=gha,mode=max
         platforms: linux/amd64
+    - 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)